src/en/proposals/dragonjspider-Wawaspecies.md

```diff
@@ -0,0 +1,64 @@
+Feel free to read all of it, but this doc will also summarise what were agreed to be good ideas, as well as the more experimental/system effecting changes. We also understand that there may be some copyright issues with this, so I will attach any permissions to this as well, If and when any are given.
+
+## Copyright permissions
+Currently we have permission from a dev of Downpour, expeditions specifically. Whilst this isn't "proper" approval, more of a semi-approval, we believe it allows us to continue with a project of making a "Scug Inspired" species. The following screenshots are provided courtesy of fairlysadpanda.
+
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/ae7437cf-f4d3-42c1-8798-b8ce24a2725f)
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/69d00309-faae-41e0-a6aa-9e2916052de3)
+
+
+
+## Species
+Our Official name for the species would be Wawa (proposed by fairlysadpanda). Whilst this isn't exactly "creative" for them, we believe a distinct but still related name suits them. We're not going to be naming them Scug(s), or Slugcat(s), because of more specific issues with copyright and such, and lack of explicit written permission.
+For personalization they will be able to have most colors for skin, similar to lizards or moths.
+Anything unmentioned will be similar to humans, like having two hands or a backpack slot.
+
+The conventional naming scheme for them, while mostly a suggestion, would be as follows: Number, Object. E.G: Seven-Hundred Nukies, Twenty-Eight Wounds, Seventy-Three Bombs. (suggested by kiterny)
+Alternatively, they could use a single word describing their character, similar to how players name lizards do, but I felt this might make them too similar in character to lizards. 
+Or finally, it could be pet names, which may be too far off what would normally be acceptable, but may be considered for names. (because of course people would name these things silly names and stuff)
+
+## Abilities
+- They will deal additional piercing damage with spears (+4 damage with spears) (suggested by thecatbastard, cornerdogs, dragonjspider)The following is the time it would take to crit a human with no resists: 
+  - 1h spear: 12 > 16 : Crits in 9 vs 7 
+  - 2h spear: 16 > 20 : Crits in 7 vs 5 
+  - 1h Reinforced spear: 15 > 19 : Crits in 7 vs 6
+  - 2h Reinforced spear: 19 > 23 : Crits in 6 vs 5 
+  - 1h Plasma spear: 18 > 22 : Crits in 6 vs 5
+  - 2h Plasma spear: 22 > 26 : Crits in 5 vs 4 
+  - 1h Uranium spear: 18 > 22 : Crits in 6 vs 5
+  - 2h Uranium spear: 22 > 26 : Crits in 5 vs 4
+
+
+* They will be small in size, similar to dwarfs, making them easier to shove down than other species.
+* They will take 20% additional damage from pierce, making Scug vs Scug combat even more lethal, and encourages a nimble playstyle rather than running face first into a gunfight.
```

Comment:
This barely affects "scug v scug"- 20% pierce is going to 99% of the time just going to make them die (a lot) faster to guns.

Reply:
Well with the whole idea with spears is what I was meaning here, with the spears doing more damage and all, I'll adjust this to clarify that. But yea the guns being more lethal was going to be one of the drawbacks of this species, maybe it's a bit too major of a drawback though...

Reply:
The 20% compounds on spear damage to reduce the amount of hits required to kill a Wawa with a spear by exactly one hit. This means that Wawa deal extra damage, and TAKE extra damage from spears, meaning if you steel the spear from them, they will die in the exact same amount of hits they'd take to kill you. That was the general idea behind it.

Reply:
Maybe, rather than a direct buff to spear damage, it can remain the same with stabbing, but thrown damage can get a x1.5 boost and they can instantly pull out spears, with no chance of them breaking, from living/dead entities? (Not walls, balance + motivates good aim) Maybe this could also be applied to everything that can be thrown and stab into people? also they should be able to throw further and have it travel faster through the air.

Reply:
Instant spear pullout is a horrible idea. That would mean you can out DPS ever melee weapon by throwing, pulling out and repeating.